pkg/util/tracing.go

```diff
@@ -0,0 +1,96 @@
+package util
```

Comment:
Considering that we can reasonably expect that this will only ever be used by pkg/global, I would suggest that this code lives there: `pkg/global/constant_rate_trace_sampler{,_test}.go` or something.

Reply:
Will do.